lib/client/jsx/components/browser/browser_pane.jsx

```diff
@@ -25,7 +43,7 @@ export default class BrowserPane extends React.Component{
       if (mode == 'edit' && !pane.attributes[attr_name].editable) return null;
 
       // The Timur view attribute.
-      let attr = pane.attributes[attr_name];
+      let attr = this.interleaveAttribute(pane.attributes[attr_name], template);
```

Comment:
Is this different than interleaveAttribute in tab_selector?

Reply:
Good catch. There is separate functionality there, but they should be merged together under `TabSelecltor.interleaveAttribute()`.